.github/workflows/production.yml

```diff
@@ -0,0 +1,67 @@
+name: Production
+
+# trigger this deployment by running:
+# gh api repos/bcgov/wps/deployments -f ref="[your branch]"
+on:
+  # reference used to figure this part out:
+  # https://docs.github.com/en/actions/reference/events-that-trigger-workflows#deployment
+  deployment:
```

Comment:
is having the `:` after `deployment` correct?

Reply:
It doesn't break it - but sure - looks weird. I've removed it.